### ISSUE TITLE
diffent output on pix2pix (U-Net)

### ISSUE THREAD

**Issue by knok** [mil-tokyo/webdnn#562]
I tried to use pix2pix with edges2cats model, but I can't get excpected output.
My implementation and models are available at:
https://github.com/knok/chainer-pix2pix
https://github.com/knok/chainer-pix2pix/releases
https://github.com/knok/chainer-pix2pix/releases/download/v0.1/models.tar.gz (pix2pix pretrained model)
https://github.com/knok/chainer-pix2pix/releases/download/v0.1/out.tar.gz (webdnn files)

![chainer output](https://knok.github.io/images/edges2cats-chainer.png)
The above is chainer output (scripts/predict.sh)

And the following image is WebDNN prediction:
![webdnn output](https://knok.github.io/images/pi2pix-webdnn.png)
You can also try on the website:
http://pix2pix.daio.net/html/index.html

I tried another dataset like edges2xxx, and they also similar output.

**Comment by Aixile** [comment]
In your webdnn model conversion script
https://github.com/knok/chainer-pix2pix/blob/master/dump_graph.py

You should set train flag of batch norm to false
```
with chainer.using_config('train', False):   
    z = enc(x)
    y = dec(z)
```

**Comment by Aixile** [comment]
I tried your models and found that the problem was solved.
![](https://i.imgur.com/YU5a93P.png)

**Comment by knok** [comment]
Thank you for your correction, I can reproduce it.
And sorry for my misunderstand.
